src/EventStoreIntegration/AggregateTranslator.php

```diff
@@ -68,6 +58,15 @@ public function extractPendingStreamEvents($anEventSourcedAggregateRoot)
     }
 
     /**
+     * @param object $anEventSourcedAggregateRoot
+     * @param Message[] $events
+     */
+    public function applyPendingStreamEvents($anEventSourcedAggregateRoot, array $events)
```

Comment:
Like discussed yesterday in the chat I think we can keep it as-is. The \Interator type hint represents the event stream and the array type hint says that the given events are not the stream but only the current pending events. 
@prolic What do you think?